fixCucumber.ps1

```diff
@@ -0,0 +1,9 @@
+If ($env:ruby_version -match "200") {
+    $path = "C:\Ruby200\lib\ruby\gems\2.0.0\gems\cucumber-2.0.0\lib\cucumber\formatter\ansicolor.rb"
+} Else {
+    $path = "C:\Ruby193\lib\ruby\gems\1.9.1\gems\cucumber-2.0.0\lib\cucumber\formatter\ansicolor.rb"
+}
+
+(Get-Content $path) -replace '\ST\DERR.puts %', '#nope' | Set-Content $path 
+(Get-Content $path) -replace '    Cucumber::', '#nope' | Set-Content $path
+(Get-Content $path) -replace '^Cucumber::', '#nope' | Set-Content $path
```

Comment:
This seems to fix a problem in Cucumber 2.0.0... is it a documented problem? Can you post a link? Does it happen with Ruby 2.2 as well?